src/lib/utils/md.ts

```diff
@@ -15,15 +15,30 @@ const getPostSlugs = (dir: string, files: string[] = []) => {
   // Temporal list of content pages allowed to be compiled
   // When a content page is migrated (and he components being used), should be added to this list
   const temporalAllowedPages = [
-    "/about",
-    "/bridges",
-    "/community/code-of-conduct",
-    "/community/events",
-    "/community/support",
+    // Use cases (7/7) ✅
```

Comment:
We certainly don't need to keep these comments, but I found them a little helpful to just keep track of all of these... figured this will be removed entirely very soon with the dev docs